app/lua53/lstate.c

```diff
@@ -42,12 +42,15 @@
 ** created; the seed is used to randomize hashes.
 */
 #if !defined(luai_makeseed)
-#if defined(LUA_USE_ESP)
+#if defined(LUA_USE_ESP8266)
```

Comment:
Isn't the `ccount` register also available on the ESP32? If so, then this should be `LUA_USE_ESP`

Reply:
`ccount` is an Xtensa thing. The ESP32-C3 (and upcoming ESP32-H2) is a RISC-V core.

Reply:
Yup but wouldn't we be introducing a new `LUA_USE_RISCV` category or at least review this hierarchy of USE source tailoring macros?